File: Models/Schedule.cs
using System;
using System.Collections.Generic;

namespace FitnessClub.Models
{
    public partial class Schedule
    {
        public int Id { get; set; }
        public int? LessonId { get; set; }
        public int? TrainerId { get; set; }
        public DateTime? BeginDate { get; set; }
        public DateTime? EndDate { get; set; }

        public FitnessLesson Lesson { get; set; }
        public Trainer Trainer { get; set; }
    }
}

File: ClientApp/src/app/Models/Schedule.ts
import { FitnessLesson } from "./FitnessLesson"
import { Trainer } from "./Trainer"

export class Schedule {
    public Id : Number
    public LessonId : Number
    public DateTime : Date
    public TrainerId: Number
    public BeginDate: Date
    public EndDate : Date

    public Lesson : FitnessLesson 
    public Trainer : Trainer

}